tests/phpunit/tests/Styles/TestHooks.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace WPML\PB\Cornerstone\Styles;
+
+use WPML\LIB\WP\PostMock;
+
+/**
+ * @group styles
+ */
+class TestHooks extends \OTGS_TestCase {
+
+	use PostMock;
+
+	public function setUp() {
+		parent::setUp();
+		$this->setUpPostMock();
+	}
+
+	/**
+	 * @test
+	 */
+	public function itShouldAddHooks() {
+		$subject = $this->getSubject();
+
+		add_action( 'save_post', [ $subject, 'invalidateStylesInTranslation' ] );
+
+		$subject->add_hooks();
+	}
+
+	/**
+	 * @test
+	 * @dataProvider dpShouldNotInvalidateStyles
+	 *
+	 * @param bool $isHandlingPost
+	 * @param bool $isTranslationEditor
+	 */
+	public function itShouldNotInvalidateStyles( $isHandlingPost, $isTranslationEditor ) {
+		$postId    = 123;
+		$metaValue = 'Some generated CSS';
+
+		update_post_meta( $postId, Hooks::META_KEY, $metaValue );
```

Comment:
@strategio I think it would be nicer to use our WP API in tests too - `Post::updateMeta` , `Post::getMetaSingle`

Reply:
Right, done!